cli/cmd/monitor.go

```diff
@@ -300,7 +303,7 @@ func AddMonitorCmd(rootCmd *cobra.Command, opts GlobalOptions) {
 		Http: &csio.RealHttpServer{},
 		Exec: &csio.RealExec{},
 	}
-	monitor.Opts.ListenAddress = monitor.Cmd.Flags().String("address", ":3000", "Custom listen address for the metrics endpoint")
+	monitor.Opts.ListenAddress = monitor.Cmd.Flags().String("address", ":3000", "Custom listen address for the metrics endpoint (not evaluated when --forward option is used)")
```

Comment:
I think it is evaluated no? It's just being used for the forwarding port, so if you do `cs monitor --forward http://localhost:8080/health --address :9000 -- ./my-app --health-runs-on :8080` you will get forwarding from localhost:9000 to localhost:8080/health